ega-data-api-filedatabase/src/main/resources/application.yml

```diff
@@ -1,20 +1,38 @@
-spring.profiles: no-oss
+spring.profiles.active: no-oss
```

Comment:
I think it should be `spring.profiles` only

Reply:
fixed